src/branch.rs

```diff
@@ -153,8 +153,7 @@ impl<'repo> Drop for Branches<'repo> {
 mod tests {
     use crate::BranchType;
 
-    #[test]
-    fn smoke() {
+    repo_test!(smoke, (Typical, TypicalWorktree, BareWorktree), |_| {
```

Comment:
I think we probably don't need to update old tests like this

Reply:
I'll revert old tests, but keep worktree tests. I'll move `repo_test` macro to `worktree.rs`